api/aggregators/searchAggregator.js

```diff
@@ -108,10 +108,26 @@ exports.createMatchAggr = async (schemaName, projectId, keywords, caseSensitive,
  */
 exports.createSortingPagingAggr = function(schemaName, sortValues, sortField, sortDirection, pageNum, pageSize) {
   const searchResultAggregation = [];
-  let defaultTwoSorts = false;
+  let dateHandling = false;
+  console.log(Object.keys(sortValues).length);
+  if (sortField && sortField.includes(",") || Object.keys(sortValues).length > 1){
+    //sort will have multiple values passed
+    if (sortField.includes("datePosted") || sortValues.hasOwnProperty("datePosted")){
```

Comment:
Is this condition only true for datePosted, or could there be issues with all dates>?

Reply:
datePosted is the only column we sort on currently and the handling is essentially just for us. We could in the future expand this to handle all types of dates if it becomes necessary.